lua/caramba/search.lua

```
@@ -858,4 +917,249 @@ M.index_workspace_keyword = function(callback)
   end)
 end
 
+-- List all index files in cache directory
+M.list_index_files = function()
+  local cache_dir = vim.fn.stdpath('cache')
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)